client/src/components/layout/AccountPage.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useEffect } from "react";
+import _ from "lodash";
+import Dropzone from "react-dropzone";
+
+const AccountPage = (props) => {
+ 
+  const [user, setUser] = useState(props.user); 
+  const [profilePicture, setProfilePicture] = useState({}); 
+  const handleImageUpload = (acceptedImage) => {
+    setProfilePicture({
+      ...profilePicture,
+      image: acceptedImage[0],
+    });
+  };
+
+  const addProfilePicture = async () => {
+    const newProfilePictureBody = new FormData();
+    newProfilePictureBody.append("image", profilePicture.image);
+    try {
+      const response = await fetch(`/api/v1/users/${profilePicture.id}`, {
```

Comment:
not sure how `profilePicture` has the `id` of the user? I would think this should instead be `user.id` to also be more readable to other developers